configure.ac

```diff
@@ -409,6 +409,9 @@ AC_MSG_NOTICE([   Configuration dir:   ${ucx_config_dir}])
 AC_MSG_NOTICE([  Preprocessor flags:   ${BASE_CPPFLAGS}])
 AC_MSG_NOTICE([          C compiler:   ${CC} ${BASE_CFLAGS}])
 AC_MSG_NOTICE([        C++ compiler:   ${CXX} ${BASE_CXXFLAGS}])
+AC_MSG_NOTICE([              CFLAGS:   ${CFLAGS}])
+AC_MSG_NOTICE([            CPPFLAGS:   ${CPPFLAGS}])
+AC_MSG_NOTICE([            CXXFLAGS:   ${CXXFLAGS}])
```

Comment:
pushed below:

```
configure: =========================================================
configure: UCX build configuration:
configure:         Build prefix:   /usr
configure:    Configuration dir:   /etc/ucx
configure:                   CC:   gcc
configure:                  CXX:   g++
configure:        BASE_CPPFLAGS:   -DCPU_FLAGS="" -I${abs_top_srcdir}/src -I${abs_top_builddir} -I${abs_top_builddir}/src
configure:          BASE_CFLAGS:   -O3 -g -Wall -Werror -fno-omit-frame-pointer -funwind-tables -Wframe-larger-than=8192 -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-label -Wno-long-long -Wno-endif-labels -Wno-sign-compare -Wno-multichar -Wno-deprecated-declarations -Winvalid-pch -Wno-pointer-sign -Werror-implicit-function-declaration -Wno-format-zero-length -Wnested-externs -Wshadow -Werror=declaration-after-statement
configure:        BASE_CXXFLAGS:   -O3 -g -Wall -Werror -fno-omit-frame-pointer -funwind-tables -Wframe-larger-than=8192 -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-label -Wno-long-long -Wno-endif-labels -Wno-sign-compare -Wno-multichar -Wno-deprecated-declarations -Winvalid-pch
configure:             CPPFLAGS:    -I/usr/include
configure:               CFLAGS:
configure:             CXXFLAGS:   -Wall
configure:           ASAN check:   no
configure:         Multi-thread:   enabled
```

Reply:
IMO unite
CPPFLAGS + BASE_CPPFLAGS
CFLAGS + BASE_CFLAGS
CXXFLAGS + BASE_CXXFLAGS

Reply:
pushed, I ordered base_cflags before cflags as they are more likely to be used in this order. tbh I prefer the previous version, in case other flags would be inserted between base_cflags and cflags this does not create false promise.

```
configure:                   CC:   gcc
configure:                  CXX:   g++
configure:             CPPFLAGS:   -DCPU_FLAGS="" -I${abs_top_srcdir}/src -I${abs_top_builddir} -I${abs_top_builddir}/src -Wall -I/usr/include
configure:               CFLAGS:   -O3 -g -Wall -Werror -fno-omit-frame-pointer -funwind-tables -Wframe-larger-than=8192 -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-label -Wno-long-long -Wno-endif-labels -Wno-sign-compare -Wno-multichar -Wno-deprecated-declarations -Winvalid-pch -Wno-pointer-sign -Werror-implicit-function-declaration -Wno-format-zero-length -Wnested-externs -Wshadow -Werror=declaration-after-statement -Wall
configure:             CXXFLAGS:   -O3 -g -Wall -Werror -fno-omit-frame-pointer -funwind-tables -Wframe-larger-than=8192 -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-label -Wno-long-long -Wno-endif-labels -Wno-sign-compare -Wno-multichar -Wno-deprecated-declarations -Winvalid-pch -Wall
configure:           ASAN check:   no
```